cdk/lib/rds-stack.ts

```diff
@@ -12,22 +14,28 @@ import { SecurityGroup, SubnetType, Vpc } from "@aws-cdk/aws-ec2";
 export interface RDSStackProps extends StackProps {
   vpc: Vpc;
   securityGroup: SecurityGroup;
+  rdsPwdSecretArnSsmParameterName: string;
 }
 
 export class RDSStack extends Stack {
-  readonly secret: ISecret;
+  public readonly rdsEndpointOutput: CfnOutput;
+  public readonly rdsUsernameOutput: CfnOutput;
+  public readonly rdsDatabaseOutput: CfnOutput;
+
   readonly postgresRDSInstance: DatabaseInstance;
   readonly rdsDbUser: string = process.env.TYPEORM_USERNAME || "serverless";
   readonly rdsDbName: string = process.env.TYPEORM_DATABASE || "awsmeetupgroup";
   readonly rdsPort: number = 5432;
-
-  constructor(scope: App, id: string, props: RDSStackProps) {
+  readonly rdsPassword: ISecret;
+  
+  constructor(scope: Construct, id: string, props: RDSStackProps) {
     super(scope, id, props);
-
-    this.secret = Secret.fromSecretAttributes(this, "rdsPassword", {
-      secretArn: `arn:aws:secretsmanager:${process.env.CDK_DEFAULT_REGION}:${process.env.CDK_DEFAULT_ACCOUNT}:secret:rdsPassword-3Eir69`,
+    
+    const secretArn = StringParameter.valueForStringParameter(this, props.rdsPwdSecretArnSsmParameterName);
+    this.rdsPassword = Secret.fromSecretAttributes(this, "rdsPassword", {
+      secretArn: secretArn
```

Comment:
nitpicky but you can reduce this to just secretArn and remove the key.

Reply:
Yeah but that hard codes the arn again, which would have to be changed in every account it gets deployed in.  Prob best to see if the partial arn search feature of CDK secrets actually works